src/commons/services/utils.js

```diff
@@ -311,6 +310,25 @@
             }
         };
 
+        vm.getTrendData = function() {
```

Comment:
why this extra method for get request when the method is already there? If it is for the testing purpose, then please don't include it in code. As this code needs to remove after the API is there.